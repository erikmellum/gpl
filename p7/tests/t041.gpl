
double x = 3.145;
double y = 3.145;

on akey
{
    x += .005;
    y += 1;
}

on skey
{
    print("x = " + x);
    print("y = " + y);
}


