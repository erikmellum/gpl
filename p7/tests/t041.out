gpl.cpp::main()
  input file(tests/t041.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  print_symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
double x 3.145
double y 3.145
gpl.cpp::main() Calling window->initialize().
gpl.cpp::main() Passing control to window->main_loop().
gpl[13]: x = 3.15
gpl[14]: y = 4.145

Printing the symbol table from signal_handler()
double x 3.15
double y 4.145
